solidity/test/random_beacon_operator/TestPricingRewardsWithdraw.js

```diff
@@ -152,28 +152,31 @@ describe('KeepRandomBeaconOperator/PricingRewardsWithdraw', function() {
 
   it("should not be able to withdraw group rewards if group is active", async () => {
     // operator1 has 3 members in group2
+    let firstActiveGroupIndex = await operatorContract.getFirstActiveGroupIndex()
+    assert.equal(firstActiveGroupIndex.toString(), 1, "Active group index should be equal to 1")
 
-    assert.isFalse(await operatorContract.isExpiredGroup('0x' + group2.toString('hex')), "Group should not be expired")
     let beneficiary1balance = web3.utils.toBN(await web3.eth.getBalance(beneficiary1))
 
     // Nothing can be withdrawn
     await expectRevert(
-      operatorContract.withdrawGroupMemberRewards(operator1, 1),
+      operatorContract.withdrawGroupMemberRewards(operator1, firstActiveGroupIndex),
```

Comment:
Can we take this opportunity and refactor this test to use:
```
const BN = web3.utils.BN
const chai = require('chai')
chai.use(require('bn-chai')(BN))
const expect = chai.expect
```

and
```
expect(...).to.eq.BN(...)
```
construction? Please see `TestTokenStakingEscrow.js` for an example.

Reply:
sure, done